### PR TITLE
add @types/tar-js

### PR DESCRIPTION
**Pull request by Narazaka**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain `{ "extends": "dtslint/dt.json" }`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson), and no additional rules.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/tar-js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Type definitions for tar-js 0.3
+// Project: http://github.com/beatgammit/tar-js
+// Definitions by: Narazaka <https://github.com/me>
```
Comment (danger-public) [inline types/tar-js/index.d.ts:3]: - :no_entry_sign: This line should have your github username in it, not /me

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## tar-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/tar-js@latest/))
was missing the following properties:
1. utils
1. stringToUint8
1. uint8ToBase64
1. base64ToUint8
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 23c055a17f6388757acd74eb52209dbef7b7e8d5
</p>

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Narazaka thx!

**Comment by orta** [comment]
👍🏻 looks good here too!

**Pull request merged**